data/schema/schema_v1.json

```diff
@@ -575,6 +575,10 @@
                                 "answer_id": {
                                   "type": "string",
                                   "description": "The id of an answer from which to obtain the max_value"
+                                },
```

Comment:
you also need to add this to the min_value

Reply:
@jonnyshaw89 Done